### ISSUE TITLE
NTSC interlaced video h264 encoding issue

### ISSUE THREAD

**Issue by Sharc99** [mltframework/shotcut#1438]
Shotcut Beta 23.04.03, Windows 11

Interlaced 4:2:2 and 4:2:0 video can't be correctly exported to NTSC 480i it seems. Chroma gets mingled between fields. 
I reported a similar issue for 576i (PAL) which got resolved (issue #1428).
Attached a .zip with a 4:2:2 and 4:2:0 interlaced source at 704x480i SD resolution.
[NTSC.ZIP](https://github.com/mltframework/shotcut/files/11409590/NTSC.ZIP)

**Comment by ddennedy** [comment]
Why did you not test this with a version *after* the fix in #1428? The fix was generic, not specific to PAL. The beta was updated with a release candidate in Releases here on GitHub, but I recommend the latest nightly build as we are releasing in a few days: https://github.com/mltframework/shotcut/actions/runs/4891243076

**Comment by ddennedy** [comment]
Please retest this with the more recent version. I am trying to follow along based on the clips and readme in the previous bug report but without AVISynth, and I am not certain there is still a problem.

I see the fix was for YUV 4:2:0 only, and your `0) 422i_NTSC.avi` does not indicate interlace or aspect ratio, which requires some workarounds to use properly in Shotcut. The workarounds depends on whether you want to fix the display aspect ratio to 4:3 or not. In either case, you need to use a custom Video Mode, matching Properties including override scan mode to interlaced, and matching Export > Video including scan mode.

**Comment by Sharc99** [comment]
I retested with RC 23.04.03 because I didn't find a newer version for download, but:
- YUV 4:2:2 works correctly after I set the aspect ratio of the 0) 422i_NTSC.avi in the properties correctly to 4:3 (or use another container which signals the aspect ratio, interlaced and field parity). Problem solved. My bad.
- YUV 4:2:0 I can't test with the current version I have, but as you fixed it based on my previous report I am confident that it will also work.
Thanks a lot.
